include/abb_librws/rws_interface.h

```diff
@@ -699,6 +699,17 @@ class RWSInterface
    */
   bool setMotorsOff();
 
+  /**
+   * \brief A method for setting the robot controller's speed ratio for RAPID motions (e.g. MoveJ and MoveL).
+   *
+   * Note: The ratio must be an integer [0, 100].
+   *
+   * \param ratio specifying the new ratio.
+   *
+   * \return bool indicating if the communication was successful or not.
```

Comment:
Only the communication, or also setting of the ratio?

Reply:
I am not 100% sure, the RWS API documentation is not very clear about it, except that it is only allowed in `Auto` mode.

I think the best way is to retrieve the speed ratio after trying to set it, and check if it has been updated, and I added a get function in 75ad57f for this purpose.

Side note, there are many functions I would like to go over and clarify the return comments for, but that will not happen this year.